server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -2835,6 +2835,126 @@ Response codes:
 
 * 204: Operation succeeded
 
+== Administrating DropLists
```

Comment:
We should add this in `manage-webadmin.md` too IMO.

But let's do that in another PR.

Reply:
Thank you! I will fix in another PR